src/main/resources/lib/nextjs/connection-config.es6

```diff
@@ -32,17 +30,15 @@ export function getSite(pathOrId, repoId) {
 exports.getFrontendServerUrl = (site) => {
     const projectName = getProjectName();
     const siteName = site?._name;
-    let url;
-    if (app.name === APP_NEXTXP_NAME) {
-        // check the app-nextjs config for project and site
-        url = app?.config?.[`settings.${projectName}.${siteName}.url`];
-    }
+
+    // check if the app is used to configure multiple sites
+    let url = app?.config?.[`nextjs.${projectName}.${siteName}.url`];
```

Comment:
```suggestion
    let url = app?.config?.nextjs?.[projectName]?.[siteName]?.url;
```

Reply:
Can't do that, because the structure is as follows:
```
app = {
  config: {
    'nextjs.<project>.<site>.url': 'url value'
    'nextjs.<project>.<site>.secret': 'secret value'    
  }
}
```